packaging/nnstreamer.spec

```diff
@@ -394,7 +394,9 @@ NNStreamer's tensor_converter and decoder subplugin of Protobuf.
 Summary:	NNStreamer Flatbuf Support
 Requires:	nnstreamer = %{version}-%{release}
 Requires:	flatbuffers
+%if "%{?profile}" != "tv"
 Recommends: flatbuffers-python
+%endif
```

Comment:
Q: With the TV profile, does NNS-flatbuffers work fine without flatbuffers-python?

Reply:
If `python script` is not used, then it's OK.